tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -438,3 +441,77 @@ async def mockup(*args):
     with running_backend.offline():
         with pytest.raises(FSBackendOfflineError):
             await alice_workspace.get_reencryption_need()
+
+
+@pytest.mark.trio
+async def test_upload_blocks(alice_user_fs, alice2_user_fs, running_backend, monkeypatch):
+    wid = await create_shared_workspace("w", alice_user_fs, alice2_user_fs)
+    alice_workspace = alice_user_fs.get_workspace(wid)
+
+    alice2_workspace = alice2_user_fs.get_workspace(wid)
+
+    fspath = "/taz"
+
+    await alice_workspace.touch(fspath)
+    await alice_workspace.sync()
```

Comment:
Should add this comment ;-)
```python
# Placeholder sync always starts by a "minimal sync" (i.e. an empty manifest is synchronized so that
# parent folder don't have to wait for the full sync before syncing itself).
# Here we force the minimal sync so to avoid this corner-case in the rest of the test.
```